changelog.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 2.7.0~rc2
+
+### Web applications
+
+* Update translation workflow to support continuous updates (#7010)
```

Comment:
Optional nit (original #7008 → backport #7010):

```suggestion
* Update CI logic for continuous localization (#7010)
```

Reply:
Since we're grouping all those in the final changelog, I'm just using the same text for now while making it clear which pr went in which release.